Aztec/Classes/Converters/NSAttributedStringToNodes.swift

```diff
@@ -575,7 +673,7 @@ private extension NSAttributedStringToNodes {
         }
 
         return unsupported.elements.reversed().flatMap({ element in
-            return element.toNode()
+            return element
         })
```

Comment:
I think we can safely remove the `flatMap`, since the `elements` collection can't contain nil values, and the closure isn't really applying any kind of modification.